api/openapi.gen.yaml

```diff
@@ -179,6 +273,24 @@ components:
         name:
           type: string
       type: object
+    v1.Reservation:
+      properties:
+        created_at:
+          format: date-time
+          type: string
+        finished_at:
+          format: date-time
+          type: string
+        id:
+          format: int64
+          type: integer
+        provider:
+          type: integer
+        status:
+          type: string
+        success:
+          type: boolean
+      type: object
```

Comment:
So success was generated here correctly (ignoring the fact this should be a pointer). Not sure why it does not generate correctly for AWS/Noop yet.